includes/Settings/SettingsUpgrade.php

```diff
@@ -304,12 +305,16 @@ public function get_extension_license_infos( $ignore_cache = false ) {
 			if ( ! empty( $bundle_upgrade_link ) && $license_status == 'valid' ) {
 				$license_info[$extension]['url'] = $bundle_upgrade_link;
 			} else {
+				$license = ! empty( $license_info[$extension]['license_key'] ) ? $license_info[$extension]['license_key'] : ( ! empty( $license_info['pro']['license_key'] ) ? $license_info['pro']['license_key'] : $license_info['templates']['license_key'] );
 				switch ( $extension ) {
 					case 'pro':
-						$license_info[$extension]['url'] = 'https://wpovernight.com/downloads/woocommerce-pdf-invoices-packing-slips-professional?utm_medium=plugin&utm_source=ips&utm_campaign=upgrade-tab&content=ips-pro-upgrade';
+						$license_info[$extension]['url'] = "https://wpovernight.com/checkout/?edd_license_key={$license}";
```

Comment:
If all three options share the same URL, the `switch` statement is no longer necessary.